packages/framework/pr_tools/trilinosprhelpers/TrilinosPRConfigurationBase.py

```diff
@@ -484,7 +484,8 @@ def pullrequest_build_name(self):
         """
         if self.arg_pullrequest_cdash_track == "Pull Request":
             output = "PR-{}-test-{}-{}".format(self.arg_pullrequest_number, self.arg_pr_genconfig_job_name, self.arg_jenkins_job_number)
-        elif self.arg_pullrequest_cdash_track == "Nightly":
+        elif self.arg_pullrequest_cdash_track == "Nightly" or \
+             self.arg_pullrequest_cdash_track == "Kokkos Integration":
```

Comment:
`elif self.arg_pullrequest_cdash_track != "Pull Request"`

Reply:
This would require all non PR track Jenkins jobs to have `DASHBOARD_BUILD_NAME` set. So this would include the Experimental track too. For now, this would only include oneapi builds.

Perhaps the better way to go about this is just:
`elif.arg_dashboard_build_name != "__UNKNOWN__"` which would cover all non-PR tracks that don't have an explicitly set `DASHBOARD_BUILD_NAME` and using the set Jenkins default value of `__UNKNOWN__`.

Future non-PR tracks can just add a value to the environment variable in jenkins without changing any code.

Reply:
Okay, so the proposed behavior is that if group is `Pull Request`, do the special first case, else if dashboard_build_name is passed and not `__UNKNOWN__`, use it, else use the genconfig string?